source/fundamentals/serialization/poco.txt

```diff
@@ -443,11 +479,36 @@ serializing the ``YearBuilt`` property if it is undefined:
    {
        public Guid Id { get; set; }
 
-       [BsonIgnore]
-       public int YearBuilt { get; set; }
+       [BsonIgnoreIfNull]
        public string Style { get; set; }
+       public int YearBuilt { get; set; }
    }
 
+You can also instruct the driver to ignore a property that contains a
+``null`` value when you register the class map, as shown in the
+following example:
+
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 4
+
+   BsonClassMap.RegisterClassMap<House>(classMap => 
+   {
+      classMap.AutoMap();
+      classMap.MapMember(h => h.Style).SetIgnoreIfNull(true);
+   });
+
+.. note:: Numerical Properties
```

Comment:
Sorry for not catching this earlier: this applies also for any [Value Type](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-types) property (for example `bool`, `char`).
"Value Type Properties" naming would be more accurate.